docs/source/existing_project.rst

```diff
@@ -1,4 +1,188 @@
+make it easier to see what changes are being made.
+
+Check your environment
+......................
+
+Make sure your system satisfies the :doc:`prerequisites <overview>`.
+
+Hydrate a copy of the template into your project
+................................................
+
+.. code:: bash
+
+    >> cd <existing_project_directory>
+    >> copier gh:lincc-frameworks/python-project-template .
+
+
+A note on directory structure
+.............................
+
+The LINCC Frameworks template does not impose a rigid directory structure. 
+However, some of the tools that the template includes expect that 
+the package source code will be in a first level directory named ``src``, and 
+that tests will be contained in a first level directory named ``tests``.
+
+For example, if your project is in a directory named ``/science``, your source 
+code and tests might look like this:
+
+.. code:: bash
+
+    /science/src/module.py
+    /science/tests/test_module.py
+
+If your directory structure for source code and tests is significantly 
+different than this, it might be worth reorganizing the files in order to make use 
+of the tools (CI, linting, auto documentation) that the template provides.
+
+While running Copier
+--------------------
+
+Reasonable responses to questions
+.................................
+If your project already has a name, especially if it's published on PyPI, it's 
+advised that you use the same name when asked "What is the name of your project?".
+
+If your project has a license already, and it is not one of the licenses listed 
+in the "What license would you like to use?", select "None".
+
+Respond **no** when asked "Do you want to create some example module code?". 
+It's fine to respond "yes", however, given that your project already has an established 
+source code and test directory structure, there's no benefit to including an example 
+module.
+
+Copier work summary
+...................
+
+After Copier has received all your answers, it will begin to hydrate the template 
+with your responses and inject them into your project. 
+As it does so, don't be scared of ``conflict`` markers - remember that everything is git tracked, 
+and Copier does not have the ability permanently overwrite your files.
+
+The following example is output from a Copier update. Note again that ``conflict`` is 
+simply an indicator that you should review that file before committing.
+
+.. code :: bash
+
+    Copying from template version 1.2.1
+    identical  .
+    identical  README.md
+    conflict  .copier-answers.yml
+    overwrite  .copier-answers.yml
+    identical  .gitignore
+    identical  .github/workflows
+    conflict  .github/workflows/linting.yml
+    overwrite  .github/workflows/linting.yml
+    identical  nb/README.md
+    conflict  .pre-commit-config.yaml
+    overwrite  .pre-commit-config.yaml
+    conflict  pyproject.toml
+    overwrite  pyproject.toml
+
+
+After running Copier
+--------------------
+
```

Comment:
Maybe also add something here about checking the diffs on conflicting or overwritten files.

Reply:
Done